src/style.css

```diff
@@ -0,0 +1,120 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+}
+
+.product-card {
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+}
+
+.product-card__image-container {
+  display: flex;
+  justify-content: center;
+}
+
+.product-card__image {
+  width: 160px;
+  height: 134px;
+}
+
+.product-card__name {
+  margin-top: 40px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.product-card__code {
+  margin-top: 4px;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+}
+
+.product-card__stars {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 16px;
+  line-height: 16px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-image: url(./images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__line :nth-child(n+5) {
+  background-image: url(./images/star.svg);
+}
+
+.product-card__rewiews {
+  font-size: 10px;
+  color: #060b35;
+}
+
+.product-card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 24px;
+}
+
+.product-card__price-name {
+  margin: 0;
```

Comment:
you could create a separate file for reset as well and reset styles for some special tags you need